magento/magento2
# Cli config show returns `Configuration ... doesn't exist` if value is 0 or empty string

### Description (*)


Fixed and covered by test.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#23290

### Manual testing scenarios (*)

1. Set any config to `0` or `''` (empty string)
2. Check value cli `bin/magento config:show {{path}}`

### Questions or comments


### Contribution checklist (*)
